### PR TITLE
Added information about CoCo4MT Workshop

### PR DESCRIPTION
**Pull request by OlenaBL**
Date, description, and topics are added.

# Description
I have added intro info about the CoCo4MT Workshop.

### CODE REVIEW

File: events/amta2022.md [hunk 1]
```diff
@@ -34,6 +34,19 @@ Students interested in MT and computational linguistics will be able to connect
 
 ## Workshops
 
+The First Workshop on Corpus Generation and Corpus Augmentation for Machine Translation (CoCo4MT) will be co-located with AMTA 2022 in Orlando, Florida, USA, on September 16th, 2022.
```
Comment (cefoo) [inline events/amta2022.md:37]: To follow our [Style Guide](http://machinetranslate.org/style) and the structure of other articles, I would apply minor stylistic changes:

> The first workshop on Corpus Generation and Corpus Augmentation for Machine Translation (**CoCo4MT**) will be co-located with AMTA 2022 in Orlando, Florida, on 16 September, 2022.

File: events/amta2022.md [hunk 2]
```diff
@@ -34,6 +34,19 @@ Students interested in MT and computational linguistics will be able to connect
 
 ## Workshops
 
+The First Workshop on Corpus Generation and Corpus Augmentation for Machine Translation (CoCo4MT) will be co-located with AMTA 2022 in Orlando, Florida, USA, on September 16th, 2022. 
+
+It is the first workshop centered around research focusing on corpora creation, cleansing, and augmentation techniques specifically for machine translation. 
+
+Topics of the workshop include but are not limited to:
```
Comment (cefoo) [inline events/amta2022.md:41]: Perhaps just:

> Possible topics:

or

> Topics (not limited):

File: events/amta2022.md [hunk 3]
```diff
@@ -34,6 +34,19 @@ Students interested in MT and computational linguistics will be able to connect
...
+- Difficulties with using existing corpora (e.g., political considerations or domain limitations) and their effects on final MT systems,
+- Strategies for collecting new MT datasets (e.g., via crowdsourcing),
+- Data augmentation techniques,
+- Data cleansing and denoising techniques,
+- Quality control strategies for MT data,
+- Exploration of datasets for pretraining or auxiliary tasks for training MT systems.
+
+More information is available at:
```
Comment (cefoo) [inline events/amta2022.md:49]: To follow the structure in other sections, let's just delete the introductory sentence and just have the link. What do you think?

**Review by cefoo — COMMENTED** [review]
Hi @OlenaBL! 🪐

Thank you so much for your PR!

I have inserted minor stylistic comments so that this section matches the structure used elsewhere. Please let me know your thoughts.

File: events/amta2022.md (hunk 1 above)
Comment (OlenaBL) [reply events/amta2022.md:37]: Sounds good!

File: events/amta2022.md (hunk 2 above)
Comment (OlenaBL) [reply events/amta2022.md:41]: Definitely.

File: events/amta2022.md (hunk 3 above)
Comment (OlenaBL) [reply events/amta2022.md:49]: Agree, thank you.

**Pull request merged**